connector_dns/README.rst

```diff
@@ -0,0 +1,60 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+    :alt: License
+
+==============
+Connector DNS
+==============
+
+This module aims to allows to manage your DNS domain through Odoo.
+
+
+Installation
+============
+
+To install this module, you need to:
+
+ * have basic modules installed (connector)
+
+
+Usage
+=====
+
+
+For further information, please visit:
+
+ * https://www.odoo.com/forum/help-1
```

Comment:
I don't think https://www.odoo.com/forum/help-1 is helpful if user has any question about this module.
Shall we put our link here?